fixtures/sub-process-no-killable

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
+'use strict';
+
+process.on('SIGTERM', () => {
+  console.log('got sigterm but we ignore it');
+});
+
+setInterval(() => {
+  // run forever
+}, 5000);
```

Comment:
You might want to increase that timeout. When run in parallel, tests can have quite some lag. See `fixtures/forever`

Reply:
Done: bb5b280c806e049600d3de4db876a7d5d6a17888